src/lib/features/feature-toggle/types/feature-toggle-strategies-store-type.ts

```diff
@@ -19,6 +19,14 @@ export interface FeatureConfigurationClient {
     variants: IVariant[];
     dependencies?: IDependency[];
 }
+
+export interface IFeatureSearchParams {
```

Comment:
This type mixes data from query params and userId so decided to create a custom type that is used by store and service